src/app/zgw/zgwClient/model/ZakenApiZaak.model.ts

```diff
@@ -0,0 +1,195 @@
+import { z } from 'zod';
+
+/**
+ * Schema en type van een zaak zoals deze in de ZGW ZakenAPI gebruik wordt.
+ * Bijvoorbeeld voor een POST, PUT of GET van een zaak
+ * Gebaseerd op versie 1.5.1 van de ZakenAPI
+ */
+const zakenApiZaakSchema = z.object({
+  identificatie: z
+    .string()
+    .max(40, 'Identificatie must be 40 characters or less.')
+    .optional(),
+
+  bronorganisatie: z
+    .string()
+    .regex(/^\d{9}$/, 'Bronorganisatie must be a 9-digit RSIN.')
+    .min(1, 'Bronorganisatie is required.')
+    .max(9, 'Bronorganisatie must be a valid RSIN.'),
+
+  omschrijving: z
+    .string()
+    .max(80, 'Omschrijving must be 80 characters or less.')
+    .optional(),
+
+  toelichting: z
+    .string()
+    .max(1000, 'Toelichting must be 1000 characters or less.')
+    .optional(),
+
+  zaaktype: z
+    .string()
+    .url('Zaaktype must be a valid URL.')
+    .min(1, 'Zaaktype is required.')
+    .max(1000, 'Zaaktype URL must be 1000 characters or less.'),
+
+  registratiedatum: z
+    .string()
+    .regex(/^\d{4}-\d{2}-\d{2}$/, { message: 'registratiedatum Datum moet in het formaat YYYY-MM-DD zijn.' })
+    .optional(),
+
+  verantwoordelijkeOrganisatie: z
+    .string()
+    .regex(/^\d{9}$/, 'VerantwoordelijkeOrganisatie must be a 9-digit RSIN.')
+    .min(1, 'VerantwoordelijkeOrganisatie is required.')
+    .max(9, 'VerantwoordelijkeOrganisatie must be a valid RSIN.'),
+
+  startdatum: z
+    .string()
+    .min(1, 'Startdatum is required.')
+    .regex(/^\d{4}-\d{2}-\d{2}$/, { message: 'startdatum Datum moet in het formaat YYYY-MM-DD zijn.' }),
```

Comment:
Waarom is deze ineens engels? Daarnaast, het is nogal random of het nederlands of engels is. ZakenApiStatus is volledig nederlands, hier is het door elkaar, is er geen standaard?